### PR TITLE
Services/biblio ref/create service

### PR DESCRIPTION
**Pull request by leogail**
Fixed Dockerfile. We'll have to fix it again because I "cheated" by copying /v1 without using .dockerignore

### CODE REVIEW

**Comment by leogail** [comment]
tests will fail : we need to set secrets without env.sh file

**Comment by parmentf** [comment]
**thought (non-blocking):** ‏Generate a `env.sh` in the actions before building

So that I don't forget.

File: services/biblio-ref/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ws-biblio-ref@0.0.0
+
+Valide une référence bibliographique
+
+Si un DOI est trouvé dans la référence bibliographique, valide la référence et indique si elle est rétractée
+
+## Prérequis
+
+Nécessite un fichier .env à la racine de biblio-ref contenant les valeurs des 3 variables WEBDAV_URL, WEBDAV_LOGIN et WEBDAV_PASSWORD
```
Comment (parmentf) [inline services/biblio-ref/README.md:9]: Donner ici un exemple de la syntaxe à utiliser dans le fichier `env.sh`.

File: services/biblio-ref/Dockerfile [hunk 2]
```diff
@@ -14,7 +14,10 @@ RUN apt update && apt -y install git curl
 # RUN dvc doctor
 # COPY ./v1/annulled.csv.dvc /dvc
 # RUN dvc pull -v
-RUN curl -v -u "$(cat /run/secrets/webdav_login):$(cat /run/secrets/webdav_password)" -o annuled.csv -O "$(cat /run/secrets/webdav_url | sed -e 's/webdavs/https/')/files/md5/e8/6afa670ff4bc8199bbee0600deb77c"
+RUN --mount=type=secret,id=webdav_login \
+    --mount=type=secret,id=webdav_password \
+    --mount=type=secret,id=webdav_url \
+    curl -v -u "$(cat /run/secrets/webdav_login):$(cat /run/secrets/webdav_password)" -o annuled.csv -O "$(cat /run/secrets/webdav_url | sed -e 's/webdavs/https/')/files/md5/e8/6afa670ff4bc8199bbee0600deb77c"
```
Comment (parmentf) [inline services/biblio-ref/Dockerfile:20]: OK, curl said that there is an authentication problem:

```log
HTTP/1.1 401 Unauthorized
```

Comment (parmentf) [reply services/biblio-ref/Dockerfile]: I juste reset the secret `WEBDAV_PASSWORD`, and now that part works.
I thought I checked this.

**Pull request merged**